packages/shared_preferences/shared_preferences/test/devtool_eval_test.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:shared_preferences_platform_interface/shared_preferences_async_platform_interface.dart';
+
+import 'shared_preferences_async_test.dart';
+
+// The tests present in this file are meant to preserve the contract
+// between shared_preferences_tool and shared_preferences. Real integration
+// tests will be implemented when the test tooling gets extracted from devtools
+// https://github.com/flutter/devtools/issues/8210
+void main() {
+  group('async', () {
+    SharedPreferencesAsync getPreferences() {
+      final FakeSharedPreferencesAsync store = FakeSharedPreferencesAsync();
+      SharedPreferencesAsyncPlatform.instance = store;
+      final SharedPreferencesAsync preferences = SharedPreferencesAsync();
+      return preferences;
+    }
+
+    // This test is meant to validate the eval performed by
+    // SharedPreferencesToolEval.fetchAllKeys
+    test('should fetch keys', () async {
+      final SharedPreferencesAsync prefs = getPreferences();
+      await prefs.setBool('test1', true);
+      await prefs.setBool('test2', true);
+      await prefs.setBool('test3', true);
+
+      // This is the actual eval, it should match the eval performed by SharedPreferencesToolEval
+      // do not edit this line unless you change the eval.
```

Comment:
Can you write all instances of the String  "do not edit <foo>" as "IMPORTANT: DO NOT EDIT <foo>"?